[2024-10-28T21:52:41.452+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-28T21:52:41.472+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_and_clean_news_sentiment_data.store_news_sentiment_data_s3 manual__2024-10-28T21:29:07.834630+00:00 [queued]>
[2024-10-28T21:52:41.476+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_and_clean_news_sentiment_data.store_news_sentiment_data_s3 manual__2024-10-28T21:29:07.834630+00:00 [queued]>
[2024-10-28T21:52:41.477+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-28T21:52:41.482+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): store_news_sentiment_data_s3> on 2024-10-28 21:29:07.834630+00:00
[2024-10-28T21:52:41.488+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=20580) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-28T21:52:41.489+0000] {standard_task_runner.py:63} INFO - Started process 20582 to run task
[2024-10-28T21:52:41.489+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fetch_and_clean_news_sentiment_data', 'store_news_sentiment_data_s3', 'manual__2024-10-28T21:29:07.834630+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/stock_data_pipelines.py', '--cfg-path', '/tmp/tmpc8rgxm72']
[2024-10-28T21:52:41.491+0000] {standard_task_runner.py:91} INFO - Job 95: Subtask store_news_sentiment_data_s3
[2024-10-28T21:52:41.504+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-28T21:52:41.532+0000] {task_command.py:426} INFO - Running <TaskInstance: fetch_and_clean_news_sentiment_data.store_news_sentiment_data_s3 manual__2024-10-28T21:29:07.834630+00:00 [running]> on host 66a068bd0545
[2024-10-28T21:52:41.589+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_and_clean_news_sentiment_data' AIRFLOW_CTX_TASK_ID='store_news_sentiment_data_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-10-28T21:29:07.834630+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-28T21:29:07.834630+00:00'
[2024-10-28T21:52:41.590+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-28T21:52:41.669+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-10-28T21:52:44.112+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-28T21:52:44.115+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-28T21:52:44.150+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=fetch_and_clean_news_sentiment_data, task_id=store_news_sentiment_data_s3, run_id=manual__2024-10-28T21:29:07.834630+00:00, execution_date=20241028T212907, start_date=20241028T215241, end_date=20241028T215244
[2024-10-28T21:52:44.209+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-28T21:52:44.227+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-28T21:52:44.229+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
